repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/ratelimiting/CheckRateLimitWConfig.groovy

```diff
@@ -54,33 +50,16 @@ class CheckRateLimitWConfig extends ReposeValveTest {
     def cleanup() {
         waitForLimitReset()
     }
-    /*
-    REP-2181 Fix JSON support for rate limits get limits call
-
-*/
 
+    //2233
```

Comment:
If this is being called out as the test of the fix for REP-2233, then it should actually say that.